configserver/src/test/java/com/yahoo/vespa/config/server/model/LbServicesProducerTest.java

```diff
@@ -127,16 +143,36 @@ public void testConfigAliasesWithRotations() throws IOException, SAXException {
                                                                                   .rotations(rotations)
                                                                                   .properties(new TestProperties().setHostedVespa(true)));
         RegionName regionName = RegionName.from("us-east-1");
-        LbServicesConfig conf = getLbServicesConfig(new Zone(Environment.prod, regionName), testModel);
-        final LbServicesConfig.Tenants.Applications.Hosts.Services services = conf.tenants("foo").applications("foo:prod:" + regionName.value() + ":default").hosts("foo.foo.yahoo.com").services(QRSERVER.serviceName);
-        assertThat(services.servicealiases().size(), is(1));
-        assertThat(services.endpointaliases().size(), is(4));
 
-        assertThat(services.servicealiases(0), is("service1"));
-        assertThat(services.endpointaliases(0), is("foo1.bar1.com"));
-        assertThat(services.endpointaliases(1), is("foo2.bar2.com"));
-        assertThat(services.endpointaliases(2), is(rotation1));
-        assertThat(services.endpointaliases(3), is(rotation2));
+        var services = getLbServicesConfig(new Zone(Environment.prod, regionName), testModel)
+                .tenants("foo")
+                .applications("foo:prod:" + regionName.value() + ":default")
+                .hosts("foo.foo.yahoo.com")
+                .services(QRSERVER.serviceName);
+
+        if (useGlobalServiceId) {
+            assertThat(services.servicealiases(), contains("service1"));
+            assertThat("Missing rotations in list: " + services.endpointaliases(), services.endpointaliases(), containsInAnyOrder("foo1.bar1.com", "foo2.bar2.com", rotation1, rotation2));
+        }
+    }
+
+    @Test
+    public void testConfigAliasesWithEndpoints() throws IOException, SAXException {
```

Comment:
I assume this test is only relevant if `!useGlobalServiceId`? In that case you can use `assumeFalse(useGlobalServiceId)` at the start of this method to avoid cluttering your test methods with `if`s. This might apply to other tests as well.

Reply:
Done!